qutebrowser/browser/hints.py

```diff
@@ -199,6 +210,12 @@ def _hint_strings(self, elems):
         Return:
             A list of hint strings, in the same order as the elements.
         """
+        if config.get('hints', 'mode') == 'words':
+            try:
+                return self._word_hinter.hint(elems)
+            except WordHintingError as e:
+                message.error(self._win_id, str(e), immediately=True)
+                # falls back on letter hints
         if config.get('hints', 'mode') == 'number':
```

Comment:
Are you sure you want this? With the current if, the hints will fall back on the "else", letter hints after giving the error message. If I make this an if-elif-else, there will be no hints at all.